apps/chat2bridge/nim.cfg

```diff
@@ -1,3 +1,4 @@
 -d:chronicles_line_numbers
 -d:chronicles_runtime_filtering:on
 -d:discv5_protocol_id:d5waku
+path = "../../waku"
```

Comment:
Me, personally would prefer better to always use the `waku/` prefix all the time we refer the library from an app.
For me it is harder to read from where are those imports coming.
WDYT?